Content.Shared/_DV/Silicons/LawElectronicsInsertionSystem.cs

```diff
@@ -0,0 +1,103 @@
+    [Dependency] private readonly SharedSiliconLawSystem _siliconLaw = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<SiliconLawProviderComponent, AfterInteractEvent>(OnAfterInteract);
+        SubscribeLocalEvent<SiliconLawProviderComponent, LawboardInsertionDoAfterEvent>(OnLawboardInsertionDoAfter);
+    }
+
+    private void OnAfterInteract(Entity<SiliconLawProviderComponent> ent, ref AfterInteractEvent args)
+    {
+        if (args.Handled || !args.CanReach || args.Target == null)
+            return;
+
+        if (!HasComp<BorgChassisComponent>(args.Target))
+            return;
+
+        if (!TryComp<WiresPanelComponent>(args.Target, out var panel) || !panel.Open)
+        {
+            if (_net.IsServer)
+            {
+                _popup.PopupCursor(Loc.GetString("lawboard-insertion-needs-panel-open",
+                        ("this", ent),
+                        ("user", args.User),
+                        ("target", args.Target)),
+                    args.User);
+            }
+
+            return;
+        }
+
+        var doAfterArgs = new DoAfterArgs(EntityManager, args.User, 10f, new LawboardInsertionDoAfterEvent(), ent, target: args.Target, used: ent)
+        {
+            NeedHand = true,
+            BreakOnDamage = true,
+            BreakOnMove = true,
+            MovementThreshold = 0.01f,
+        };
+
+        if (!_doAfter.TryStartDoAfter(doAfterArgs))
+            return;
+
+        _popup.PopupPredicted(
+            Loc.GetString("lawboard-insertion-start-actor-message", ("board", ent), ("target", args.Target)),
+            Loc.GetString("lawboard-insertion-start-other-message", ("board", ent), ("target", args.Target), ("actor", args.User)),
+            args.Target.Value,
+            args.User);
+    }
+
+    private SiliconLawset GetLawset(ProtoId<SiliconLawsetPrototype> lawset)
+    {
+        var proto = _prototype.Index(lawset);
+        var laws = new SiliconLawset()
+        {
+            Laws = new List<SiliconLaw>(proto.Laws.Count)
+        };
+        foreach (var law in proto.Laws)
+        {
+            laws.Laws.Add(_prototype.Index<SiliconLawPrototype>(law));
+        }
+        laws.ObeysTo = proto.ObeysTo;
+
+        return laws;
+    }
+
+    private void OnLawboardInsertionDoAfter(Entity<SiliconLawProviderComponent> ent, ref LawboardInsertionDoAfterEvent args)
+    {
+        if (args.Handled || args.Cancelled || args.Args.Target is not {} target)
+            return;
+
+        var newLaws = GetLawset(ent.Comp.Laws).Laws;
+
+        if (!TryComp<SiliconLawProviderComponent>(target, out var targetComp))
+            return;
+
+        if (targetComp.Lawset == null)
+            targetComp.Lawset = new SiliconLawset();
+
+        targetComp.Lawset.Laws = newLaws;
+        _siliconLaw.NotifyLawsChanged(target);
```

Comment:
delete the law board and play magin.ogg or something after

Reply:
not in the design

Reply:
for AI a law board is used when swapping its laws, this just copies it for free
may as well not have a cost for evil board if you can just apply it to every borg for free